pkgdb/src/fetchers/wrapped-nixpkgs-input.cc

```diff
@@ -97,12 +97,7 @@ createWrappedFlakeDirV0( const nix::FlakeRef & nixpkgsRef )
   flox::NixState           nixState;
   nix::ref<nix::EvalState> state = nixState.getState();
   nix::FlakeRef wrappedRef = nix::parseFlakeRef( "path:" + tmpDir.string() );
-  /* Push verbosity level to suppress "warning: creating lock file ..." */
-  auto oldVerbosity = nix::verbosity;
-  nix::verbosity    = nix::lvlError;
-  auto _locked      = flox::lockFlake( *state, wrappedRef, {} );
-  /* Pop verbosity */
-  nix::verbosity = oldVerbosity;
+  auto          _locked    = nix::flake::lockFlake( *state, wrappedRef, {} );
```

Comment:
Do you plan to add back the verbosity backup/restore parts?

Its getting kinda difficult to say when to use `flox::` functions vs `nix::flake::` functions.
It probably makes sense when looking at the call "tree" but is generally easy to do wrong, and tricky to debug.
Since the flox variants are necessary, is there something we can do for them to be safer to use as a general replacement for `nix::flake::*`?

Reply:
I'm not sure about the verbosity.  Seems like it was there for a reason, to prevent an unexpected warning, but it really hindered the debugging.  Something to review before merging for sure.

There may be a better approach to fixing the original issue so it may not be as necessary.  The fear is that we're playing wack a mole with this kind of thing (trying to make sure the nix download thread doesn't get started in the parent), and it might be better to see if there's a different approach to fixing the original problem.

IMO, since this is blocking the release, if there's a way to make it work, that we understand to some extent with as few changes (and risk) as possible, that would be a good step to take ASAP for the release.  That will buy us some time to revisit the mechanism and see if there's a better way.  But as for making it safer... there might be a way, I'll look at that now.